cluster-autoscaler/cloudprovider/azure/azure_scale_set_test.go

```diff
@@ -94,7 +112,7 @@ func TestTargetSize(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
-	expectedScaleSets := newTestVMSSList(3, "test-asg", "eastus")
+	expectedScaleSets := newTestVMSSList(3, "test-asg", "eastus", compute.Uniform)
```

Comment:
IMO, wherever we had tests for uniform mode, there should be a test with mode = flex